openprescribing/templates/faq.html

```diff
@@ -0,0 +1,114 @@
+{% extends "base.html" %}
+{% load template_extras %}
+
+{% block title %}API{% endblock %}
+{% block active_class %}more{% endblock %}
+
+{% block content %}
+
+
+<h1>FAQs</h1>
+
+<p>We strongly recommend reading NHS Digital's <a href="http://content.digital.nhs.uk/gpprescribingdata">introduction to prescribing data</a>, including their excellent <a href="http://content.digital.nhs.uk/media/10048/FAQs-Practice-Level-Prescribingpdf/pdf/PLP_FAQs_April_2015.pdf">FAQ</a> (PDF) and <a href="http://content.digital.nhs.uk/media/10686/Download-glossary-of-terms-for-GP-prescribing---presentation-level/pdf/PLP_Presentation_Level_Glossary_April_2015.pdf">glossary of terms</a> (PDF).</p>
+
+<p>Is your question not listed below? Send us a message <a href="mailto:{{ SUPPORT_EMAIL }}" class="doorbell-show">here</a>!</p>, and we'll get back to you</p>
+
+<p><strong>General</strong></p>
+<p><a href="#siteupdate">When do you update the site?</a></p>
+<p><a href="#extractdownload">How do I download an extract of the data?</a></p>
+<p><a href="#cite">How do I cite OpenPrescribing?</a></p>
+<p><a href="#timeperiod">What time period does the dataset cover?</a></p>
+
+<p><strong>Prescribing data</strong></p>
+<p><a href="#prescitem">What is a prescription item?</a></p>
+<p><a href="#prescquantity">What does quantity mean?</a></p>
+<p><a href="#actualcost">What is actual cost?</a></p>
+<p><a href="#bnfchange">How do you deal with BNF code changes?</a></p>
+
+<p><strong>Analyse</strong></p>
+<p><a href="#analysetutorial">How do I use the analyse page?</a></p>
+<p><a href="#denominator">What denominator should I use?</a></p>
+<p><a href="#appliance">How can I search for appliance data?</a></p>
+
+<p><strong>CCG & GP Dashboards</strong></p>
+<p><a href="#measureinterpret">How do I interpret the measures?</a></p>
+<p><a href="#measurechose">How are the measures chosen?</a></p>
+<p><a href="#gpclose">A GP Practice has closed or merged - why is it included on your dashboards?</a></p>
+
+<p><strong><a href="#Disclaimer">Disclaimer</a></strong></p>
+
+<hr/>
+
+<h3>General</h3>
+
+<p><strong><a id="siteupdate">When do you update the site?</a></strong></p>
```

Comment:
I would make each FAQ section a header (e.g. `<h3>`) rather than a link, as making them a link makes it look like you can click on them, but you can't.

So instead of

    <p><strong><a id="siteupdate">When do you update the site?</a></strong></p>

I would do

    <h3 id="siteupdate">When do you update the site?</h3>